content/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

```diff
@@ -87,6 +87,10 @@ Terraform Cloud expects the team names in the team membership SAML attribute to
 
 You can configure SSO Team IDs in the organization's **Teams** page. If an SSO Team ID is configured, Terraform Cloud will attempt to match the chosen SAML attribute against both the team name and the SSO Team ID when mapping users to teams. You may want to create an SSO Team ID if the team membership SAML attribute is not human readable and is not used as the team's name in Terraform Cloud.
 
+SSO Team IDs are particularly helpful if your SSO or Active Directory provider restricts the `MemberOf` attribute in its SAML responses to Group UUIDs, rather than human readable group names. Setting the SSO Team ID allows you to maintain human readable team names in Terraform Cloud, while still managing team membership through SSO or AD.
+
+![Screenshot: SSO Team ID configuration](/img/docs/sso-team-id-config.png)
```

Comment:
I'd prefer that we remove this screenshot. The reason is that our UI changes often, and screenshots are difficult to maintain, so I want to limit their use to when we absolutely need them. Are you okay with that? Push back if you think it's critical here!

Reply:
I agree, I was kinda 50/50 on this screen shot to begin with. I was thinking it would be helpful if the user has seen the field in the UI and was looking for an explanation as to what it does, but I guess if they have arrived at this section of the docs they probably have their answer already.

Plus there was a spelling mistake in it, so this gives me an excuse to erase that embarrassment from history 🙂